examples/otlpmetrics-function/collector-deployment/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM otel/opentelemetry-collector-contrib:0.87.0
```

Comment:
This isn't a huge issue, but it would be good to use a more recent (e.g. 0.110.0 version of the collector)

Reply:
@dashpole PTAL: #380